TCAT/AppDelegate.swift

```diff
@@ -105,6 +108,104 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     
     // MARK: Helper Functions
     
+    /// Convert BusStop and PlaceResult models to new unified Place model.
+    func migrationToNewPlacesModel() {
+        
+        // Potentially check version to make sure this only fires once.
+        
+        //
+        // allStops
+        //
+        
+        // should overwrite existing values
+        // getBusStops()
+        
+        //
+        // favorites
+        //
+        
+        let favoritesKey = Constants.UserDefaults.favorites
+        
+        if let storedPlaces = userDefaults.value(forKey: favoritesKey) as? Data {
+            // This will only fire on legacy models
+            if let favorites = NSKeyedUnarchiver.unarchiveObject(with: storedPlaces) as? [Any] {
```

Comment:
nitpick: can we make this one if let like `if let storedPlaces = ___, let favorites = ____ { ... }`